### ISSUE TITLE
Installation on Btrfs partition fails

### ISSUE THREAD

**Issue by Frosch96** [calamares/calamares#693]
Hello,
I cannot install on a Btrfs partition. I always get the following error message:

> Failed to unpack image /bootmnt/chakra/x86_64/root-image.sqfs
> rsync failed with error code 11

```
17:44:50 [0]: Starting job "mount" 
mount: unbekannter Dateisystemtyp „Btrfs“
mount: unbekannter Dateisystemtyp „Btrfs“
17:44:50 [0]: Starting job "unpackfs" 
mount: /bootmnt/chakra/x86_64/root-image.sqfs is used as read only loop, mounting read-only
rsync: write failed on "/tmp/calamares-root-9ik51ej8/usr/lib/firmware/brcm/brcmfmac43362-sdio.bin": No space left on device (28)
rsync error: error in file IO (code 11) at receiver.c(393) [receiver=3.1.2]
17:46:33 [0]: Installation failed: 
17:46:33 [0]: - message: "Failed to unpack image /bootmnt/chakra/x86_64/root-image.sqfs" 
17:46:33 [0]: - details: "rsync failed with error code 11." 
17:46:44 [0]: Calamares will now quit. 
17:46:44 [8]: Shutting down Calamares... 
17:46:44 [8]: Finished shutdown. 
17:46:44 [0]: QBasicTimer::start: QBasicTimer can only be used with threads started with QThread
17:46:44 [0]: QBasicTimer::start: QBasicTimer can only be used with threads started with QThread
```

**Comment by teo** [comment]
Could you please paste the full terminal output of `calamares -d` while reproducing this issue?
Also the contents of your partition.conf and mount.conf could be relevant.

**Comment by adriaandegroot** [comment]
Given this bit of the provided log: `mount: unbekannter Dateisystemtyp „Btrfs“`, I'm going to think that what happens is the filesystem is specified wrongly with a capital B, mount(8) doesn't mount anything, and then the filesystem is just another directory in /tmp, which fills up rapidly.

This has been fixed in 6f680c0269f092a05507841071386d93ed9e05bd (with specific mention of Fat32, but it's a general lower-casing issue).
